src/main/java/org/folio/des/service/impl/JobServiceImpl.java

```diff
@@ -190,6 +201,19 @@ public void deleteJobs(List<Job> jobs) {
     jobExecutionService.deleteJobs(jobs);
   }
 
+  @Override
+  public InputStream downloadExportedFile(UUID jobId) throws Exception {
+    Job job = getJobEntity(jobId);
+    if (job.getFiles().isEmpty()) {
+      throw new Exception("The URL of the exported file is missing");
```

Comment:
Lets introduce new runtime FileDownloadException and used it in all places instead of Exception and RuntimeException